Makefile

```diff
@@ -56,29 +66,55 @@ docker-from-scratch:
 docker-compile-env:
 	docker build -f build/Dockerfile.interactive-compile-environment -t authservice-build-env:$(USER) .
 
-bazel-bin/src/main/auth_server:
-    # Note: add --compilation_mode=dbg to the end of the next line to build a debug executable with `make docker-from-scratch`
+$(current_binary):
+# Note: add --compilation_mode=dbg to the end of the next line to build a debug executable with `make docker-from-scratch`
```

Comment:
I tried `make docker` after this PR seems got some error. maybe the docker rule needs update as well? (`fly` is my linux user name)

```
mkdir -p /home/fly/workspace/github.com/istio-ecosystem/authservice/.cache/tools/bazelisk /home/fly/workspace/github.com/istio-ecosystem/authservice/.cache/bazel
GOARCH=amd64 /usr/local/go/bin/go run github.com/bazelbuild/bazelisk@v1.11.0 --output_user_root=/home/fly/workspace/github.com/istio-ecosystem/authservice/.cache/bazel build   //src/main:auth_server
make: *** No rule to make target 'fly', needed by 'docker'.  Stop.
```

Reply:
Got it will update.

Reply:
I changed this to `make image`. I will deal with everything container in a follow up PR if you don't mind.

Reply:
np.